src/store/actions.js

```diff
@@ -136,8 +137,8 @@ export default{
                     authors[0].link = result.authors.author.link;
                 }
                 Vue.set(ResultsObj,'authors', authors);
-                Vue.set(ResultsObj,'similar_books',(typeof result.similar_books == 'object' ? result.similar_books.book : ""));
-              
+                //Vue.set(ResultsObj,'similar_books',(typeof result.similar_books == 'object' ? result.similar_books.book : ""));
+                ResultsObj.similar_books = getSimilarBooks();
```

Comment:
to set an array to a new value in Vue you have to use  ` Vue.set(Obj,'arr',newarr); `
`Vue.set(ResultsObj,'similar_books',getSimilarBooks()); `
this should work